orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -21,6 +21,9 @@ using namespace onnxruntime::logging;
 using namespace onnxruntime::training;
 
 struct TrainingParameters {
+  std::string model_with_loss_function_path;
```

Comment:
these are some values i found useful to hardcode in this file for debugging. exposing them to Python would at least allow them to be set without rebuilding ORT, but i'm not sure how/if they should be exposed from ORTTrainer.